lib/rules/prefer-module-scope-constants.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        'Prefer that screaming snake case variables always be defined using `const`, and always appear at module scope.',
+      category: 'Best Practices',
+      recommended: true,
+    },
+    schema: [],
+  },
+
+  create(context) {
+    let inConstDeclaration = false;
+
+    return {
+      VariableDeclaration(node) {
+        inConstDeclaration = node.kind === 'const';
+      },
+      VariableDeclarator(node) {
+        const {id} = node;
+
+        if (id.type !== 'Identifier' || id.name !== id.name.toUpperCase()) {
+          return;
+        }
+
+        if (!inConstDeclaration) {
+          context.report(
+            node,
+            'You must use `const` when using screaming snake case variables. If this is not a constant, use camelcase instead.'
```

Comment:
`camelcase` -> `camelCase`?

Reply:
`using` => `defining`.

Reply:
@kaelig I think `camelcase` is the correct way to spell it in a sentence